menus/menu-main.sh

```diff
@@ -6,6 +6,9 @@ then
   echo
   exit 0
 fi
+
+APPSETTINGSTYPE='NA'
```

Comment:
Just thinking we could put this in app-constant-reset? 
It would remove logic from a menu script and would also keep all the resets together.